geosys/ui/widgets/geosys_coverage_downloader.py

```diff
@@ -58,7 +60,15 @@
 CVIN_THUMBNAIL_URL = (
     '{bridge_url}/field-level-maps/v4/season-fields/{id}/coverage/{image}'
     '/base-reference-map/INSEASON_CVIN/thumbnail.png')
-
+YGM_THUMBNAIL_URL = (
```

Comment:
We need to move these including the `CVIN_THUMBNAIL_URL` to the definitions file.

Reply:
@Samweli I can move these to default, I just stayed with the original structure on where these URLs are stored in this py file - only NDVI_THUMBNAIL_URL existed at that time, so the rest has been added by me when the catalog-imagery changes has been required. Will move these for you.